util/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+#   Copyright © 2020 by The qTox Project Contributors
+#
+#   This file is part of qTox, a Qt-based graphical interface for Tox.
+#   qTox is libre software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   qTox is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with qTox.  If not, see <http://www.gnu.org/licenses/>
```

Comment:
I'm glad we're making sure that if anyone writes `add_library(util_library STATIC ${HEADER_LIST})` in their cmake files, they will be creating a derived work of qtox and have all their stuff under GPLv3.

Reply:
This was also in the global cmake file before, I see no reason or justification to drop that?